README.md

```diff
@@ -1 +1,235 @@
-# terraform-google-bigquery-dataset-iam
\ No newline at end of file
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+
+[![Terraform Version][badge-terraform]][releases-terraform]
+[![Google Provider Version][badge-tf-gcp]][releases-google-provider]
+[![Join Slack][badge-slack]][slack]
+
+# terraform-terraform-google-bigquery-dataset-iam
+
+A [Terraform] module for [Google Bigquery IAM][google-bigquery-iam] on [Google Cloud Platform (GCP)][gcp].
+
+**_This module supports Terraform version 1
+and is compatible with the Terraform Google Provider version 3._**
+
+This module is part of our Infrastructure as Code (IaC) framework
+that enables our users and customers to easily deploy and manage reusable,
+secure, and production-grade cloud infrastructure.
+
+- [Module Features](#module-features)
+- [Getting Started](#getting-started)
+- [Module Argument Reference](#module-argument-reference)
+  - [Top-level Arguments](#top-level-arguments)
+    - [Module Configuration](#module-configuration)
+    - [Main Resource Configuration](#main-resource-configuration)
+    - [Extended Resource Configuration](#extended-resource-configuration)
+- [Module Attributes Reference](#module-attributes-reference)
+- [External Documentation](#external-documentation)
+- [Module Versioning](#module-versioning)
+  - [Backwards compatibility in `0.0.z` and `0.y.z` version](#backwards-compatibility-in-00z-and-0yz-version)
+- [About Mineiros](#about-mineiros)
+- [Reporting Issues](#reporting-issues)
+- [Contributing](#contributing)
+- [Makefile Targets](#makefile-targets)
+- [License](#license)
+
+## Module Features
+
+A [Terraform] base module for creating a `terraform_google_bigquery_iam_*` resource. This module creates a google bigquery IAM with additional feature to either IAM binding or member and binds into a role depending on `authoritative` variable.
```

Comment:
please review before opening the resource is not `terraform_google_bigquery_iam_`

```suggestion
A [Terraform] base module for creating a `google_bigquery_dataset_iam_*` resources.

It allows authoritative bindings (exclusive setting members), non-authoritative (adding additional members), or policy based IAM management of resource level access.
```